### PR TITLE
Remove rows where loc id is duplicated

### PR DESCRIPTION
**Pull request by james-francis-MT**
# Description
remove locations where location id is duplicated for the same import date

# How to test
Currently testing in branch - will update

https://trello.com/c/9to9YH3t/183-epic-5-remove-all-rows-where-the-locationid-is-duplicated-in-ascwds-workplace-file-must

# Developer checklist
- [ ] Unit test
- [ ] Linked to Trello ticket
- [ ] Documentation up to date

### CODE REVIEW

**Review by garypolzin — CHANGES_REQUESTED** [review]
the function looks alright but it needs adding to the clean_ascwds_workplace_data script please, not this one :)

File: jobs/clean_ascwds_workplace_data.py [hunk 1]
```diff
@@ -66,6 +69,20 @@ def cast_to_int(df: DataFrame, column_names: list) -> DataFrame:
...
+def remove_locations_with_duplicates(df: DataFrame):
+    loc_id_import_date_window = Window.partitionBy(
+        AWP.location_id, PartitionKeys.import_date
+    )
+
+    df_with_count = df.withColumn(
+        "location_id_count", F.count(AWP.location_id).over(loc_id_import_date_window)
+    )
+
+    df_without_duplicates = df_with_count.filter(F.col("location_id_count") == 1)
```
Comment (luke-a-williams) [inline jobs/clean_ascwds_workplace_data.py:81]: Your tests explain what you are trying to achieve here very well.

**Review by luke-a-williams — APPROVED** [review]
GTM

File: tests/unit/test_clean_cqc_location_data.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import warnings
```
[the remark this answers is not in the document]
Comment (james-francis-MT) [reply tests/unit/test_clean_cqc_location_data.py:2]: Just splitting them up so you can run each set of tests for each function

**Pull request merged**